R/get_schema.R

```diff
@@ -57,9 +57,24 @@ schema_exists <- function(conn, schema) {
   checkmate::assert_class(conn, "DBIConnection")
   checkmate::assert_character(schema)
 
+  if (inherits(conn, "SQLiteConnection")) return(FALSE)
+
   objs <- DBI::dbListObjects(conn)
   matches <- sapply(objs$table, \(.x) methods::slot(.x, "name")) |>
     (\(.x) names(.x) == "schema" & .x == schema)()
 
-  return(any(matches))
+  if (any(matches)) return(TRUE)
+
+  tryCatch({
+    DBI::dbCreateTable(
+      conn,
+      name = DBI::Id(schema = schema, table = "schema_test"),
```

Comment:
Consider using a more specific name for the table.
Like "SCDB_schema_test"
(inspired by the dbplyr_XXX) naming schema for temporary tables.

Reply:
The name is completely irrelevant in my opinion; the call is only considered if there are no tables within the schema in question, and the table is deleted shortly after. The only reason I gave it a name is because you cannot create a table with an empty name.

Reply:
I went about and made the changes anyways 🙂